DataFormats/SiStripCluster/src/classes_def.xml

```diff
@@ -1,12 +1,19 @@
 <lcgdict>
 
- <class name="SiStripCluster" ClassVersion="13">
+ <class name="SiStripCluster" ClassVersion="14">
+  <version ClassVersion="14" checksum="1374720584"/>
   <version ClassVersion="13" checksum="1374720584"/>
   <version ClassVersion="12" checksum="2984011925"/>
   <version ClassVersion="11" checksum="3702468681"/>
   <version ClassVersion="10" checksum="3791198690"/>
   <field name="error_x" transient="true"/>
  </class>
+ <ioread sourceClass="SiStripCluster" version="[10]" targetClass="SiStripCluster" source="std::vector<uint8_t> amplitudes_; uint16_t firstStrip_;" target="barycenter_,charge_" >
+   <![CDATA[SiStripCluster::initQB_(onfile.amplitudes_, onfile.firstStrip_, charge_, barycenter_);]]>
+ </ioread>
+ <ioread sourceClass="SiStripCluster" version="[11-13]" targetClass="SiStripCluster" source="std::vector<uint8_t> amplitudes_; uint16_t firstStrip_; float barycenter_;" target="barycenter_,charge_" >
+   <![CDATA[if (onfile.barycenter_ == 0) SiStripCluster::initQB_(onfile.amplitudes_, onfile.firstStrip_, charge_, barycenter_);]]>
+ </ioread>
```

Comment:
@cms-sw/core-l2 
this compiled for me, but the old data with FWLite `Tree::Scan` was crashing with a segfault.
So, I decided to push anyways to discuss

Something that I think may be unusual is that I updated the class version to 14 without changing the checksum (since the class data layout did not change).

... I tried to change the layout by adding a new data member, but for some reason there was no reaction from the reflex: the dictionary compiled no matter what I had in the version fields (e.g. using the old version 13 with the old checksum.
`scram b updateclassversion` or a regular scram build compiles without any complaints.
The log has 
`Skipped running EDM Class Version test`. Is this expected?